src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -110,7 +112,8 @@ const HeatmapPlot = (props) => {
   }, [heatmapSettings]);
 
   useEffect(() => {
-    const selectedGenesLoading = _.intersection(selectedGenes, loadingGenes).length > 0;
+    const selectedGenesLoading = _.intersection(selectedGenes, loadingGenes).length > 0
+      && fetchingGenes;
```

Comment:
Fix the heatmap sometimes showing loader when the embedding is loading genes for itself (not for the heatmap)